drivers/wifi/nrf700x/Kconfig

```diff
@@ -401,3 +401,26 @@ config NRF_WIFI_BEAMFORMING
 config WIFI_NRF700X_SCAN_TIMEOUT_S
 	int "Scan timeout in seconds"
 	default 30
+
+config NRF_WIFI_2G_BAND
+	bool ""
+	default y if BOARD_NRF7001
+
+config NRF_WIFI_5G_BAND
```

Comment:
Why the empty help text? I guess this is the user facing option, no?

Reply:
Yes, Adding help text.